components/button/button-move.js

```diff
@@ -49,11 +51,41 @@ class ButtonMove extends FocusMixin(RtlMixin(LitElement)) {
 			 * @type {boolean}
 			 */
 			disabled: { type: Boolean, reflect: true },
```

Comment:
Is there a purpose in still having `disabled` with all the fine-grained control down below? How do these new attributes interact with it and `disabled-tooltip`?

Reply:
Maybe I should compute `disabled` for the `button` based on these `disabled-*`, and ditch the `disabled-tooltip` until we have a concrete need for it.

Reply:
Ok, got this all cleaned up!